SpatialGDK/Source/SpatialGDKServices/Private/SpatialCommandUtils.cpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialCommandUtils.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialCommandUtils);
+
+bool SpatialCommandUtils::AttemptSpatialAuth(bool IsRunningInChina)
+{
+	FString SpatialInfoArgs = IsRunningInChina ? TEXT("auth login --environment=cn-production") : TEXT("auth login");
+	FString SpatialInfoResult;
+	FString StdErr;
+	int32 ExitCode;
+
+	const static FString SpatialExe = TEXT("spatial");
+	FPlatformProcess::ExecProcess(*SpatialExe, *SpatialInfoArgs, &ExitCode, &SpatialInfoResult, &StdErr);
```

Comment:
Do we have access to `SpatialGDKServicesConstants::SpatialExe`?
Or is that not in this release.
If we don't - please pass TEXT("spatial") directly.

Reply:
In 0.6 we don't have this in a nice place. We do in 0.8 though.

Reply:
Can you pass TEXT("spatial") directly? No need for a static.